src/js/lotto.js

```diff
@@ -0,0 +1,24 @@
+import { LOTTO } from './constants/index.js';
+
+const getLottoNumbers = (money) => {
+  return money / LOTTO.PRICE;
+};
+
+const generateLotto = () => {
+  const lotto = [];
+  while (lotto.length < LOTTO.LENGTH) {
+    const number = Math.floor(Math.random() * LOTTO.MAX_NUM) + 1;
+
+    if (lotto.includes(number)) continue;
+    lotto.push(number);
+  }
+  return lotto;
+};
```

Comment:
```suggestion
const lottoNumbers = Array(LOTTO.MAX_NUM).fill(1).map((v, k) => v + k);
const generateLotto = () => lottoNumbers.sort(() => 0.5 - Math.random()).slice(0, LOTTO.LENGTH);
```
이런식으로 표현할 수 있답니다!
![image](https://user-images.githubusercontent.com/18749057/179386842-f8fba49a-18dd-4636-adc2-1c129a154a9e.png)

그리고 정렬까지 해주면 더 좋겠네요 ㅎㅎ

```suggestion
const lottoNumbers = Array(LOTTO.MAX_NUM).fill(1).map((v, k) => v + k);
const generateLotto = () => lottoNumbers.sort(() => 0.5 - Math.random()).slice(0, LOTTO.LENGTH).sort((a, b) => a - b);
```